omas/machine_mappings/d3d.py

```diff
@@ -724,4 +779,9 @@ def magnetics_probes_data(ods, pulse):
 
 
 if __name__ == '__main__':
-    test_machine_mapping_functions(__all__, globals(), locals())
+    test_machine_mapping_functions(['langmuir_probes_data'], globals(), locals())
+
+
+'''
+
+'''
```

Comment:
Why an empty docstring at the end?

Reply:
That's just cruft, I need to remove it. I am waiting on #159 to merge this. Unless you're in a hurry